Pipfile

```diff
@@ -100,9 +105,9 @@ SQLAlchemy-Utils = "==0.32.21"
 SQLAlchemy = "==1.2.2"
 thrift_sasl = "==0.3.0"
 pipenv = "*"
-python-ldap = "==3.1.0"
```

Comment:
Why we have removed this lib?

Reply:
due to duplicacy in  Pipfile i.e. it was mentioned 2 times in same Pipfile